bedrock/privacy/templates/privacy/notices/firefox-rocket.html

```diff
@@ -0,0 +1,43 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It seems this new template is identical to `firefox-quantum.html`, which is identical to `firefox-cliqz-quantum.html`. Should we DRY things up a bit here by making a new base for these pages to inherit from?

Reply:
Good idea. It really is just a new base template. I'll work on that.